examples/dishwasher-app/silabs/include/DeviceEnergyManager.h

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Ideally this should be a common file not Silabs specific?

Reply:
After review of how the EVSE app is doing its init, I think this file is better of as App Specific. I was even considering merging it into the Dishwasher manager, currently keeping it bu in the long run we might remove it.